torch/_inductor/codegen/rocm/ck_universal_gemm_template.py

```diff
@@ -231,9 +255,14 @@ def emit_ck_instance(self, op: "CKGemmOperation"):
         template_params = []
         for field_name, field_value in op.dict_items():
             if isinstance(field_value, tuple):
-                template_params.append(
-                    f"/* {field_name} */ S<{', '.join(map(str, iter(field_value)))}>"
-                )
+                if "ds" in field_name:  # element type and layout for bias
+                    template_params.append(
+                        f"/* {field_name} */ Tuple<{', '.join(map(str, iter(field_value)))}>"
+                    )
+                else:  # tile shape
+                    template_params.append(
+                        f"/* {field_name} */ S<{', '.join(map(str, iter(field_value)))}>"
```

Comment:
Hoist the `', '.join(map(str, iter(field_value)))`? And maybe `template_params.append`.